sections/04_study.md

```diff
@@ -82,7 +82,101 @@ use interesting network architectures to approach single-cell data.
 
 ### Metagenomics
 
-*@gailrosen will write this*
+*@gailrosen will write this:*
+	Metagenomics (which refers to the study of genetic material, 16S rRNA 
+and/or whole-genome shotgun DNA, from microbial communities) has 
+revolutionized the study of micro-scale ecosystems within us and around us. 
+There is increasing literature of applying machine learning in general to 
+metagenomic analysis.  In the late 2000’s, a plethora of machine learning 
+methods were applied to classifying DNA sequencing reads to the thousands of 
+species within a sample.  An important problem is genome assembly from these 
+mixed-organism samples. And to do that, the organisms should be “binned” 
+before assembling.  Binning methods began with many k-mer techniques [refs] 
+and then delved into other clustering algorithms, such as self-organizing maps 
+(SOM).  Then came the taxonomic classification problem,  with researchers 
+naturally using BLAST [blast], followed by other machine learning techniques 
+such as SVMs [McHardy], naive Bayesian classifiers [nbc], etc. to classify 
+each read.  Then, researchers began to use techniques that could be used to 
+estimate relative abundances of an entire sample, instead of the precise but
+painstakingly slow read-by-read classification.  Relative abundance
+estimators (a.k.a diversity profilers) are MetaPhlan[ref], (WGS)Quikr[ref],
+and some configurations of tools like OneCodex[ref] and LMAT[ref].  While one
+cannot identify which reads were mapped back to an organism using relative
+abundance estimators, they can be useful for faster comparative and other
+downstream analyses.   Newer methods hope to classify reads and estimate
+relative abundances at faster rates [Vervier] and as of this writing, there
+are more than 70 metagenomic taxonomic classifiers in existence.  Besides
+binning and classification of species, there is functional identification and
+annotation of sequence reads [Yok,Soueidan]. However, the focus on
+taxonomic/functional annotation is just the first step.  Once organisms are
+identified, there is the interest in understanding the interrelationship
+between these organisms and host/environment phenotypes [Guetterman].  One of
+the first attempts was a survey of supervised classification methods for
+microbes->phenotype classification [Knights], followed by similar studies
+that are more massive in scale [Stratnikov, Segata].  There have been
+techniques that bypass the taxonomic classification step altogether [ Ding et al]
+, (sequence composition to phenotype classification).  Also, researchers have
+looked into how feature selection can improve classification [Liu, Segata],
+and techniques have been proposed that are classifier-independent
+[Ditzler,Ditzler].
+
+So, how have neural networks (NNs) been of use?    Most neural networks are being 
```

Comment:
How much improvement is gained with deep networks? Do they allow researchers any additional information that  more traditional ML approaches do not?

Perhaps this paragraph and the next could be combined with the next two since it looks like they discuss method improvement in detail with specific examples.